wire-ios-request-strategy/Sources/Payloads/Processing/ConversationEventPayloadProcessor.swift

```diff
@@ -81,6 +81,17 @@ struct ConversationEventPayloadProcessor {
             Logging.eventProcessing.error("Conversation creation missing timestamp in event, aborting...")
             return
         }
+        guard let conversationID = payload.id ?? payload.qualifiedID?.uuid else {
+            Flow.createGroup.fail(ConversationEventPayloadProcessorError.noBackendConversationId)
+            Logging.eventProcessing.error("Conversation creation missing conversationID in event, aborting...")
+            return
+        }
+        guard await context.perform({
+            ZMConversation.fetch(with: conversationID, domain: payload.qualifiedID?.domain, in: context) == nil
+        }) else {
```

Comment:
Does it make sense to add a unit test for this scenario? In case of refactorings or maintenance it might help in future :)

Reply:
It's possible but there's currently no unit tests operating at this level so that's why I hesitated to go down that path. I'll take another look and see how big of effort it would be.

Reply:
I've added the missing unit tests